packages/notification-services-controller/src/NotificationServicesController/types/on-chain-notification/schema.ts

```diff
@@ -1,4 +1,5 @@
-/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable jsdoc/tag-lines */
+/* eslint-disable jsdoc/check-tag-names */
```

Comment:
Adding these since this schema was auto-generated from an openAPI spec.
This reduces additional work required when we need to update this openAPI spec.